examples/proto/examplepb/echo_service.swagger.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Echo Service",
+    "title": "examples/proto/examplepb/echo_service.proto",
```

Comment:
This seems like an odd change. Why did this become the file path?

Reply:
I will remove it but the strange thing is that is part of the autogenerated code so I don't know why it is changed.

Reply:
Actually, I if remove that line, the build process fails https://circleci.com/gh/grpc-ecosystem/grpc-gateway/1982?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
The title of the documents was kept due to some changes when generating the swagger documentation.